### PR TITLE
Dialect Hotkeys

### PR DESCRIPTION
**Pull request by eduardobbgf**

### CODE REVIEW

File: modules/apps/product-navigation/product-navigation-applications-menu-web/src/main/resources/META-INF/resources/js/ApplicationsMenu.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ import '../css/ApplicationsMenu.scss';
 const getOpenMenuTooltip = (keyLabel) => (
 	<>
 		<div>{Liferay.Language.get('open-menu')}</div>
-		<kbd className="c-kbd c-kbd-dark">
+		<kbd className="c-kbd">
```
Comment (ethib137) [inline modules/apps/product-navigation/product-navigation-applications-menu-web/src/main/resources/META-INF/resources/js/ApplicationsMenu.js:34]: This file should not have been changed.

File: modules/apps/frontend-theme/frontend-theme-dialect/src/WEB-INF/frontend-token-definition.json [hunk 2]
```diff
@@ -4751,33 +4750,8 @@
 							],
 							"name": "hotkeysLgLineHeight",
```
Comment (ethib137) [inline modules/apps/frontend-theme/frontend-theme-dialect/src/WEB-INF/frontend-token-definition.json:4751]: Following the pattern in other components like Buttons, the individual token should be in the singular form. ie: hotkeyLgLineHeight

**Review by ethib137 — CHANGES_REQUESTED** [review]
Great work! I'm really happy with how much you all have accomplished with this.

We will go over all the comments on the call today so you both can have a chance to ask questions about my suggestions. 

- The hotkeys frontend token set should probably be broken into 2 separate ones: `hotkeys` and `general`. 
- Any time we run a command to generate code, such as buildLang, we should include the generated code in it's own commit. This makes it easy to update the non generated code and then regenerate the generated code and merge commits. 

Again, I'm really happy with how much you both have learned. Good job.

**Comment by ilzamcmed** [comment]
Just starting reviewing :)

**Comment by ilzamcmed** [comment]
Just starting reviewing :)

**Pull request closed without merging**